packages/core/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./node_modules/@hcengineering/platform-rig/profiles/default/tsconfig.json",
 
   "compilerOptions": {
+    "target": "ES2019",
```

Comment:
May be we should change it in platform-rig/profiles/default/tsconfig.json?

Reply:
changed